doc/telescope.txt

```diff
@@ -636,6 +636,11 @@ builtin.man_pages({opts})                                *builtin.man_pages()*
     Parameters: ~
         {opts} (table)  options to pass to the picker
 
+    Fields: ~
+        {sections} (table)  a list of sections to search, use `{ "ALL" }` to
+                            search in all sections
+
+
```

Comment:
Please dont manually edit `doc/telescope.txt` its autogenerated. You need to change it here https://github.com/nvim-telescope/telescope.nvim/blob/b7cd8c7699623c4b351009192dc9a03a54195e6b/lua/telescope/builtin/init.lua#L243-L245
Note: currently our parser can only handle single line annotations. Limitation i am already working on.
Also you should read CONTRIBUTING.md :)

Other stuff LGTM  thanks :)